internal/exec/engine.go

```diff
@@ -76,6 +76,14 @@ func (e Engine) Run(stageName string) bool {
 
 	e.Logger.PushPrefix(stageName)
 	defer e.Logger.PopPrefix()
+
+	if e.Root != "/" {
```

Comment:
Can we move the previous definition of the base config into here? That way, we can just populate the path correctly the first time.